packages/react-cards/src/components/Card/Card.view.tsx

```diff
@@ -95,3 +97,18 @@ export const CardView: ICardComponent['view'] = props => {
     </Slots.root>
   );
 };
+
+function _isReactElement(item: React.ReactNode): item is React.ReactElement {
+  return !!item && typeof item === 'object' && !!(item as React.ReactElement).type;
+}
+
+function _isCardItem(item: React.ReactNode): item is typeof CardItem {
+  // In theory, we should be able to just check item.type === CardItemType.
+  // However, under certain unclear circumstances (see https://github.com/OfficeDev/office-ui-fabric-react/issues/10785),
+  // the object identity is different despite the function body being the same.
```

Comment:
Just a question, should we also have this comment in the `_isCardSection` method? I know it would be repeating the same info but I've seen this before in other files. I don't have a strong opinion on this either, just wanted to bring it up.

Reply:
I decided to omit it since it's only a few lines down from the same comment in _isCardItem